.travis/install_cmsgemos.sh

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+
+# Travis script to install cmsgemos
+
+cd $TRAVIS_BUILD_DIR/..
+git clone --depth 1 https://github.com/cms-gem-daq-project/cmsgemos.git
```

Comment:
I see...so everytime someone makes a PR this is going to clone `cmsgemos`.  So bugs in `cmsgemos` might cause a unit test to this repo to fail?  One needs to then keep this in mind when interpreting results.

Reply:
Right. We could specify a specific revision (or tag?), but would have to update it manually.

Reply:
No let's keep it this way and rely the HEAD of cmsgemos being correct